packages/squiggle-lang/package.json

```diff
@@ -7,6 +7,8 @@
     "bundle": "tsc && webpack",
     "start": "rescript build -w -with-deps",
     "clean": "rescript clean",
+    "coverage": "jest --coverage",
```

Comment:
(`"scripts":` section of `package.json` is going to be very different once https://github.com/QURIresearch/squiggle/pull/62 merges in, expect MC) 

I tried `jest`'s coverage report and didn't like it-- it talks about `.bs.js` files, not `.res` files. See solution in https://github.com/QURIresearch/squiggle/issues/65